public-ui/src/app/views/pm/pm.component.css

```diff
@@ -76,7 +76,7 @@ button {
 }
 
 .participant-count {
-    margin-left: 30%;
+    margin-left: 35%;
```

Comment:
Looks better with this.

Reply:
try: 
    width: 100%;
    text-align: center;
    padding-bottom: 18px; 

I will put the count in the middle and give a little more padding.

Reply:
Done. thanks.